src/Integration/Connection/UI/ConnectionInfoDialogViewModel.cs

```diff
@@ -37,6 +38,7 @@ internal class ConnectionInfoDialogViewModel : NotifyErrorViewModelBase
         } = true;
         private string serverUrlRaw;
         private string username;
+        private OrganizationInformation selectedOrganization;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'selectedOrganization'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1144)